### PR TITLE
Closes #119: Add person demo content.

### LINKED ISSUE #119: Person content type demo content

### PR DESCRIPTION
**Pull request by trackleft**
Reuse ua_quickstart person demo content, and provide a general demo content rubrik.

## Description
As started in the wednesday workshop, adding demo content for az person from ua_quickstart.
Adds:
People images as az_media
Person categories
Person secondary categories
Person content

## Related Issue
#119 

## How Has This Been Tested?
locally
https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-812.probo.build/people

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

## Checklist:
- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by joeparsons** [comment]
Double trouble on the migration test site (as expected):
https://0ea6a403-258f-45ab-85bc-c2bf5c0255de--site-migration.probo.build/people

**Comment by camikazegreen** [comment]
This looks awesome.

**Review by camikazegreen — APPROVED** [review]

**Review by mmunro-ltrr — APPROVED** [review]
Still needs Rosalind Hardy–Weinberg (professor in EEB), but that's waiting on a placeduck!

**Review by camikazegreen — DISMISSED** [review]
The primary and secondary taxonomy tags are now displaying correctly on the probo build.

File: modules/custom/az_demo/data/az_demo_menu_links.json [hunk 1]
```diff
@@ -21,6 +21,39 @@
       "expanded": false,
       "enabled": true,
       "weight": "2"
+    },
+    {
+      "link_id": "3",
+      "parent_link_id": "0",
+      "menu": "main",
+      "title": "People",
+      "urlpath": "<nolink>",
```
Comment (trackleft) [inline modules/custom/az_demo/data/az_demo_menu_links.json:30]:
```suggestion
      "urlpath": "route:<button>",
```

**Review by mmunro-ltrr — APPROVED** [review]

**Review by smayden — APPROVED** [review]

**Comment by camikazegreen** [comment]
This looks great.

Not directly related to this PR, but...
Did we make an intentional decision to not separate by taxonomy on the default person views in QS2?
Compare:
- Kitten: https://kitten.ltrr.arizona.edu/people
- Probo: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-812.probo.build/people

**Comment by smayden** [comment]
> This looks great.
> 
> Not directly related to this PR, but...
> Did we make an intentional decision to not separate by taxonomy on the default person views in QS2?
> Compare:
> 
> * Kitten: https://kitten.ltrr.arizona.edu/people
> * Probo: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-812.probo.build/people

Are they separated by default in QS1? Or just how they are shown on Kitten? 

I can't remember if we discussed specifically, but I would not want the list of people separated unless a content editor specifically set it up that way (using the View). 

But perhaps for the demos site, we will want to add a page that separates out by taxonomy term to demonstrate/test?

**Comment by trackleft** [comment]
> > This looks great.
> > Not directly related to this PR, but...
> > Did we make an intentional decision to not separate by taxonomy on the default person views in QS2?
> > Compare:
> > 
> > * Kitten: https://kitten.ltrr.arizona.edu/people
> > * Probo: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-812.probo.build/people
> 
> Are they separated by default in QS1? Or just how they are shown on Kitten?
> 
> I can't remember if we discussed specifically, but I would not want the list of people separated unless a content editor specifically set it up that way (using the View).
> 
> But perhaps for the demos site, we will want to add a page that separates out by taxonomy term to demonstrate/test?

I was wondering similar things.

**Review by tadean — APPROVED** [review]

**Pull request merged**